Alloy/MTLPixelFormat+CoreGraphics.swift

```diff
@@ -0,0 +1,80 @@
+import Metal
+
+public extension MTLPixelFormat {
+
+    enum CGAlphaInfo {
+        case noneSkip
+        case premultiplied
+    }
+
+    var compatibleCGColorSpace: CGColorSpace? {
```

Comment:
I don't think ColorSpace can be inferred anyhow from pixel format, definetely should be user-provided instead